pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -186,8 +186,8 @@ func (e *Controller) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer e.queue.ShutDown()
 
-	klog.Infof("Starting endpoint controller")
-	defer klog.Infof("Shutting down endpoint controller")
+	klog.InfoS("Starting Endpoint controller")
+	defer klog.InfoS("Shutting down Endpoint controller")
```

Comment:
if you're capitalizing "Endpoint" because it's a type, then the actual type name is "Endpoints" with an "s", not "Endpoint"

Reply:
Fixed.